packages/eslint-plugin-pf-codemods/src/rules/v6/chartsImportMoved/chartsImportMovedOutput.tsx

```diff
@@ -0,0 +1 @@
+import { Chart } from '@patternfly/react-charts/victory';
```

Comment:
Nit: for the output files we like to have them as the 1:1 output of running the fixer, which here would have Chart on a new line.

The reasoning for this is that in the near future we will have a CI check that makes sure the output matches the input after autofixes are applied.

Reply:
I don't really understand why Chart would be on a new line. I had to fix all that for Cost Management's linter.